README.md

```diff
@@ -19,7 +19,9 @@ ReactDOM.render(React.createElement(MyApp), document.getElementById('app'));
 AppRegistry.registerComponent('MyApp', () => MyApp);
 ```
 
-This is where your application enters into the React domain and comes alive.
+This is where your application enters into the React domain and comes alive. Your
+root React element gets created and attached to some existing element in the host
+environment ('app' in this case).
```

Comment:
I think this is too specific to be correct. Can you simplify this to `Your root React element is instantiated and attached to the host environment.`?